ui/models/browsertimeAlertsExtraData.js

```diff
@@ -112,6 +112,32 @@ export default class BrowsertimeAlertsExtraData {
           artifactPath: `public/test_info/profile_${suite}.zip`,
         });
 
+        // We check the artifacts with `method: 'HEAD'` which means that it
+        // doesn't download the whole file and only gets the headers. We can see
+        // if artifact is available or not using the headers.
+        const promise = fetch(url, {
```

Comment:
Interesting, I didn't know about this function. But unfortunately `getData` doesn't work for us for this case because it's doing more work than what we need here.

After getting the response with fetch, it's trying to parse the json body here:
https://github.com/mozilla/treeherder/blob/eccd74e32ea098bd5b57a1695bc1ff142ca49606/ui/helpers/http.js#L32
And since we are requesting the data with `method: HEAD`, the response that is returned doesn't have a `body`. Because of that this `await response.json()` line is throwing an exception that says:
> Uncaught (in promise) SyntaxError: JSON.parse: unexpected end of data at line 1 column 1 of the JSON data

So that's why we can't use `getData`, otherwise I will need to handle this error.